firmware/quadruna/VC/src/app/states/app_pcmState.c

```diff
@@ -0,0 +1,82 @@
+#include "app_pcmState.h"
+#include "app_allStates.h"
+#include "app_hvInitState.h"
+#include "app_initState.h"
+
+#include "app_canUtils.h"
+#include "app_canTx.h"
+#include "app_canRx.h"
+
+#include "app_powerManager.h"
+#include "app_pumpControl.h"
+#include "app_faultCheck.h"
+
+static PowerStateConfig power_manager_pcm = {
+    .efuses = {
+        [EFUSE_CHANNEL_SHDN] = true,
+        [EFUSE_CHANNEL_LV] = true,
+        [EFUSE_CHANNEL_PUMP] = false,
+        [EFUSE_CHANNEL_AUX] = false,
+        [EFUSE_CHANNEL_INV_R] = true,
+        [EFUSE_CHANNEL_INV_L] = true,
+        [EFUSE_CHANNEL_TELEM] = true,
+        [EFUSE_CHANNEL_BUZZER] = false,
+    },
+};
+
+static void pcmStateRunOnEntry(void)
+{
+    app_canTx_VC_State_set(VC_PCM_STATE);
+    app_powerManager_updateConfig(power_manager_pcm);
+
+    // TODO: enable PCM
+    // io_pcm_set(true);
+
+    // Disable inverters and apply zero torque upon entering pcm state
+    app_canTx_VC_LeftInverterEnable_set(true);
```

Comment:
I am gonna read into this but I am wondering if we should be holding off on enabling the inverters until PCM is ready. Because like at the same time inverters dont depend on PCM but I am just wondering in terms of logic